src/plugins/dashboard/public/dashboard_app/dashboard_app.test.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import React from 'react';
+import { createMemoryHistory } from 'history';
+import { DashboardApp } from './dashboard_app';
+import { render } from '@testing-library/react';
+import { buildMockDashboard } from '../mocks';
+import * as dashboardApiHelpers from '../dashboard_container/external_api/dashboard_api';
+
+/* These tests circumvent the need to test the router and legacy code
+/* the dashboard app will be passed the expanded panel id from the DashboardRouter through mountApp()
+/* @link https://github.com/elastic/kibana/pull/190086/
+*/
+describe('Dashboard App', () => {
```

Comment:
Can we clean up all of the `An update to ForwardRef inside a test was not wrapped in act(...).` warnings that get thrown in this test? 😅